apimodels/cedar_models.go

```diff
@@ -5,6 +5,7 @@ type CedarConfig struct {
 	RPCPort  string `json:"rpc_port"`
 	Username string `json:"username"`
 	APIKey   string `json:"api_key,omitempty"`
+	Insecure bool   `json:"tls"`
```

Comment:
This json name likely to be confusing, because tls: true sounds like the opposite of insecure: true

Reply:
This is "insecure" everywhere else.

Reply:
Nice catch!